src/app/components/Search/index.tsx

```diff
@@ -154,13 +156,26 @@ const SearchCmp: FC<SearchProps> = ({ scope, variant, disabled, onFocusChange: o
   const searchButtonContent =
     variant !== 'button' ? <SearchIcon sx={{ color: COLORS.grayMediumLight }} /> : t('search.searchBtnText')
 
-  const errorMessage = isTooShort
-    ? t('search.error.tooShort')
-    : hasPrivacyProblem
+  const errorMessage = isTooShort ? t('search.error.tooShort') : undefined
+  const hasError = !!errorMessage
+
+  const warningMessage = hasPrivacyProblem
     ? t('search.error.privacy', { appName: t('appName'), wordsOfPower })
     : undefined
+  const hasWarning = !!warningMessage
 
-  const hasError = !!errorMessage
+  const hasProblem = hasError || hasWarning
+
+  useEffect(() => {
+    if (hasProblem) {
+      const timeout = setTimeout(() => {
+        setItsTimeToAct(true)
+      }, 500)
```

Comment:
(I dont like this pattern much. I'd validate either instantly, or on submit)

Reply:
Actually we validate instantly in the sense that the search button is disabled right away; we just don't want to shove the error message to the face of the user. (Hopefully he will keep typing and won't even have to see it at all.)